### ISSUE TITLE
Mace Stun Proc

### ISSUE THREAD

**Issue by Maadguy** [RetroWoW/RetroWoW#212]
Mace Stun proc rate is way too high, it procs basically every time and can litteraly stunlock players down.

**Comment by Alicca** [comment]
True... if it happens sometimes we can call it lucky but it happens a lot for such low % to proc
